polaris-react/src/components/Button/Button.scss

```diff
@@ -359,6 +359,7 @@
           rgba(255, 255, 255, 0.1) 0%,
           rgba(255, 255, 255, 0) 100%
         );
+        // stylelint-disable-next-line polaris/conventions/polaris/custom-property-allowed-list -- se23 temporary styles
```

Comment:
I think this might be a needless disable ([lint log](https://github.com/Shopify/polaris/actions/runs/5469257435/jobs/9957959845?pr=9622#step:8:137))

Reply:
Strange, it was passing locally with the disable and failing without 🙃